test/controllers/shared/basic_rest_response_test.rb

```diff
@@ -47,30 +49,14 @@ def basic_pagination_per_page_test
         test 'should render correct per_page value' do
           get :index, params: {per_page: @entries_per_page + 1}, session: set_session_user
           assert_response :success
-          per_page_results = css_select('.pagination-pf-items-current')
-          assert_equal "1-#{@entries_per_page + 1}", per_page_results.first.content.squish
+          per_page_results = response.body.scan(/perPage":\d+/).first.gsub(/[^\d]/, '').to_i
+          assert_equal @entries_per_page, per_page_results
         end
 
         test 'should render per page dropdown with correct values' do
           get :index, params: {per_page: @entries_per_page + 1}, session: set_session_user
           assert_response :success
-          assert_select "[id='per_page']" do
-            assert_select "option[selected='selected'][value='#{@entries_per_page + 1}']"
-            assert_select "option[value='5']"
-            assert_select "option[value='10']"
-            assert_select "option[value='15']"
-            assert_select "option[value='25']"
-            assert_select "option[value='50']"
-          end
-        end
-
-        test 'should display pagination buttons' do
```

Comment:
why is this test irrelevant?

Reply:
There is no HTML of the React component here, only the ruby mounter.
I think we can count on Patternfly to test their components, especially displaying the pagination buttons..